applications/tari_merge_mining_proxy/src/main.rs

```diff
@@ -66,7 +66,7 @@ use tokio::runtime::Runtime;
 
 pub type PrivateKey = RistrettoSecretKey;
 
-pub const LOG_TARGET: &str = "base_node::conduit_app";
+pub const LOG_TARGET: &str = "tari_mm_proxy";
```

Comment:
```suggestion
pub const LOG_TARGET: &str = "tari_mm_proxy::main";
```
We might have more than one file in the logging of the application later.

Reply:
Have updated this.